### DISCUSSION TITLE
question DM

### DISCUSSION THREAD

**Discussion by magarthur** [mp2i-fsm/mp2i-2021#124] [Question] [answered]
Bonsoir Monsieur,

Pour le DM, sommes-nous obliger d'écrire tous les algorithmes dans des fichiers différents ou pouvons-nous par exemple créer un fichier "list" (en .ml), et dans ce fichier avoir différents algorithmes comme la taille d'une liste, renverser une liste... ?

Cordialement,

Arthur

**Comment by fortierq** [comment, marked as answer]
Bonjour,

Il est préférable de regrouper les algorithmes similaires ensemble dans le même fichier. 
Sur https://github.com/fortierq/mp2i-library j'écris : 

> Vous pouvez regrouper les fonctions similaires dans le même fichier (exemple : toutes les fonctions de base sur les listes dans list.ml).

Après je vous laisse voir comment vous souhaitez regrouper exactement.
